### ISSUE TITLE
instability Wifi PrinCube

### ISSUE THREAD

**Issue by stefano71** [dukelec/mb#6]
I don't understand why the Wifi of my PrinCube keeps saying Wrong Password with both iPhone and Android. I proceeded to upgrade the firmware version PrinCube-fw-4.26_4.11.tar but the same problem occurs. It only works with the PC but there is no way to connect the wifi to the phone.

I'm waiting for your answer

**Comment by dukelec** [comment]
Sorry, as I said in last release notice, it should be a hardware problem, please contact the after-sales service first, provide the order number and a short video of the connection failure, they will give you a new device. If it is not resolved, I will follow up.

**Comment by stefano71** [comment]
please give me a contact to replace the defective product

**Comment by dukelec** [comment]
May I ask where you purchased it?

**Comment by Greenslider** [comment]
Hey, will the hardware mod with the capacitors fix the wifi problem

**Comment by dukelec** [comment]
@Greenslider 
Only the oldest machine needs the capacitor replacement: https://github.com/dukelec/mb/releases/tag/4.26_4.11
Can you take a picture of the PCB? It is best to also provide the machine CPU ID (for example, via USB connection). I will confirm whether it needs to be replaced.

**Comment by Greenslider** [comment]
Yeah no problem, the ID what i found over USB was:  3b4a6c820080070114192a0a
and attached is the photo of the pcb
![1608801393085](https://user-images.githubusercontent.com/54683981/103078098-79ddea00-45d1-11eb-93f6-ee6d2c9689da.jpg)

**Comment by dukelec** [comment]
@Greenslider This board is indeed old and the capacitor needs to be replaced.

**Comment by Greenslider** [comment]
@dukelec Okey, thank you for youre fast reply, can you help me wich size the capacitor are? And should ther be some capacitor in the circled area, where only one component is soldered?

**Comment by dukelec** [comment]
@Greenslider 
The capacitances in the circle are not the most critical, so you can leave them alone.

You only need to replace two 5.6pF capacitors. The capacitor footprint is 0402. It is recommended to choose a higher capacitor accuracy (e.g. ±0.1pF).

**Comment by stefano71** [comment]
I also have the old card and I have no way to change the condenser. can you send a new card?

**Comment by Greenslider** [comment]
Hey, today i replaced the capacitors and everything works now perfect. Thank you so much for your help
